NSS-Day-Cohort-53/tabloid-full-stack-always-picturesque-icebergs
# Fs edit posts

#### Changes Made

1. Added file CreatePost to Components/CreatePost directory.
1.     Modified file PostRepository to include Update and functionality.
2.     Modified file PostController to include Put and functionality.
3.     Modified file PostManager to include updatePost and functionality.
4.     Modified file EditPost to include functionality.
5.     Modified file ApplicationViews to render EditPost.
6.     Modified file Post to show Edit Post option.


#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout fs-EditPosts
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > WHEN user clicks on the edit link on a Post Details page, THEN they are directed to the edit form.
    > WHEN user clicks save on the edit form, THEN the post is updated in teh database.
    > WHEN user clicks cancel on the edit form, THEN they are returned to the previous page.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
